feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableGrid.kt

```diff
@@ -330,13 +335,17 @@ private data class TimetableItemLayout(
     val minutePx: Float,
     val dayToStartTime: MutableMap<DroidKaigi2023Day, Instant>,
 ) {
+    val tz = TimeZone.of("UTC+9")
     val dayStart = dayToStartTime[timetableItem.day] ?: dayStartTime
+    val gridStartTime = LocalTime(hour = 10, minute = 0)
+    val gridStart =
```

Comment:
[Help wanted ⚠️]
I am not familiar with how to use date time APIs so much.
I think it is quite dirty and there should be better ways, so I would like some advice to improve this calculation 🙏

Reply:
Ref: Initial time is 10:00 https://github.com/DroidKaigi/conference-app-2023/blob/main/feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridHours.kt#L271